src/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- fontawesome -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="style.css">
+  <title>MovieTV</title>
+</head>
+<body>
+  <header>
+    <h1>MovieShow</h1>
+    <nav>
+      <div class="desktop-menu">
+        <ul>
+          <li id="all">All Movie <span class="counter movies"></span></li>
+          <li>planets</li>
+          <li>Races</li>
```

Comment:
[OPTIONAL] I don't think you should include this since there are not doing anything.